src/diverter.cc

```diff
@@ -0,0 +1,91 @@
+#include "cyclus.h"
+#include "pyre.h"
+#include "process.h"
+#include "diverter.h"
```

Comment:
Google Style suggested iclude order:
`Use standard order for readability and to avoid hidden dependencies: Related header, C library, C++ library, other libraries' .h, your project's .h.`